test/all.dart

```diff
@@ -24,7 +24,7 @@ void defineTests() {
       DartToJsScriptRewriter transformer = _transformer();
       Document document = new Document.html(_html);
       transformer.rewriteDartTags(document);
-      List dartJsScripts = document.querySelectorAll('script').where((tag) {
+      var dartJsScripts = document.querySelectorAll('script').where((tag) {
```

Comment:
Understood. If we're running analyzer as part of the CI process, this will be caught I think.